src/__tests__/fireEvent.test.js

```diff
@@ -193,6 +193,46 @@ test('should not fire on disabled Pressable', () => {
   expect(handlePress).not.toHaveBeenCalled();
 });
 
+test('should not fire on disabled TextInput', () => {
+  const placeholder = 'Test placeholder';
+  const onChangeTextMock = jest.fn();
+  const NEW_TEXT = 'New text';
+
+  const { getByPlaceholderText } = render(
+    <View>
+      <TextInput
+        editable={false}
+        placeholder={placeholder}
+        onChangeText={onChangeTextMock}
+      />
+    </View>
+  );
+
+  fireEvent.changeText(getByPlaceholderText(placeholder), NEW_TEXT);
+  expect(onChangeTextMock).not.toHaveBeenCalled();
+});
+
+test('should not fire on disabled TextInput with nested Text', () => {
+  const placeholder = 'Test placeholder';
+  const onChangeTextMock = jest.fn();
+  const NEW_TEXT = 'New text';
+
+  const { getByPlaceholderText } = render(
+    <View>
```

Comment:
we could add the custom `TextInputWrapper` here and combine all cases into a single test :)